jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/AbstractHTWorkItemHandler.java

```diff
@@ -60,6 +60,8 @@ public abstract class AbstractHTWorkItemHandler implements WorkItemHandler {
     
     protected static final String ADMIN_USER = System.getProperty("org.jbpm.ht.admin.user", "Administrator");
     
+    protected static final Boolean COPY_TASK_SUBJECT_TO_Task_DESCRIPTION = Boolean.parseBoolean(System.getProperty("org.jbpm.ht.copy.task_subject.to.task_description", "true"));
```

Comment:
if this is not need outside of this class, I would change the access modifier to `private`. Also, make sure the name is completely uppercase.

```suggestion
    private static final Boolean COPY_TASK_SUBJECT_TO_TASK_DESCRIPTION = Boolean.parseBoolean(System.getProperty("org.jbpm.ht.copy.task_subject.to.task_description", "true"));
```

Reply:
corrected typo and access modifier.